webapp/src/shared/components/Row/Row.tsx

```diff
@@ -0,0 +1,58 @@
+import Icon, { IconVariant, IconSize } from '../Icon/Icon';
+import './Row.css';
+import { TextColor, TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+import { SmallAvatar } from '../Avatar/Avatar';
+import { StatusVariant } from '../Status/Status';
+
+type RowProps = {
+  iconVariant?: IconVariant;
+  onClick?: () => void;
+  avatarUrl?: string;
+  avatarStatus?: StatusVariant;
+  title?: string;
+  subtitle?: string;
+};
+
+export default function Row({
+  iconVariant,
+  onClick,
+  avatarUrl,
+  avatarStatus,
+  title,
+  subtitle,
+}: RowProps) {
+  return (
+    <button className={`row paragraph-regular`} onClick={onClick}>
+      {avatarUrl &&
+        ((avatarStatus && (
+          <SmallAvatar url={avatarUrl} status={avatarStatus} />
+        )) || <SmallAvatar url={avatarUrl} />)}
```

Comment:
I do this because otherwise we have this:
![Screenshot from 2022-07-18 01-48-50](https://user-images.githubusercontent.com/2774420/179427922-f6352552-cb8e-4d5a-8dc1-14de32182128.png)

```
<figure class="avatar-small__image-wrapper avatar-status--undefined">...</figure>
```
Which doesn't seem very safe as a className to me. But maybe it's worth it to have simpler logic, and strange classNames that will probably (although not necesarilly) not exist.

Reply:
Ah look, we have this classname anyway...